generator/controller_template.go

```diff
@@ -169,6 +189,21 @@ func (c *{{.schema.ID}}Controller) AddClusterScopedHandler(ctx context.Context,
 	})
 }
 
+func (c *{{.schema.ID}}Controller) AddClusterScopedFeatureHandler(enabled func() bool, ctx context.Context, name, cluster string, handler {{.schema.CodeName}}HandlerFunc) {
```

Comment:
Is there a specific case this satisfies? 
Can a user enable a feature on a per cluster basis?